cocos/particle/renderer/particle-system-renderer-data.ts

```diff
@@ -203,11 +203,24 @@ export default class ParticleSystemRenderer {
 
     private _particleSystem: any = null!; // ParticleSystem
 
+    create (ps: any) {
+        if (this._particleSystem === null) {
```

Comment:
Need to emit error when create is invoked and `_particleSystem` is not null, notify user when they are not invoking APIs correctly, not allowing create to be invoked twice with different particle system.